index.js

```diff
@@ -235,33 +234,35 @@ function schemaMiddleware (schema) {
 }
 
 function cors (opts) {
-  var _cors = corsify(opts)
+  opts = opts || {}
+  var headers = opts.headers || ['Content-Type', 'Accept', 'X-Requested-With']
+  var methods = opts.methods || ['PUT', 'POST', 'DELETE', 'GET', 'OPTIONS']
```

Comment:
Ah, just saw you're assuming this behavior in the tests. While it works, it means every request is wasting extra bytes which can add up quickly in regular HTTP because the headers aren't compressed. I think optimizing makes a lot of sense here.